baselines/diabetic_retinopathy_detection/dropout_tune_on_retention_auc.py

```diff
@@ -298,11 +313,12 @@ def step_fn(inputs):
     logging.info(message)
 
     # Run evaluation on all evaluation datasets, and compute metrics
-    total_results = utils.evaluate_model_and_compute_metrics(
+    per_pred_results, total_results = utils.evaluate_model_and_compute_metrics(
       strategy, eval_datasets, steps, metrics, eval_estimator,
       uncertainty_estimator_fn, per_core_batch_size, available_splits,
-      estimator_args=estimator_args, is_deterministic=False,
-      num_bins=FLAGS.num_bins, use_tpu=use_tpu)
+      estimator_args=estimator_args, call_dataset_iter=False,
```

Comment:
Note that we set call_dataset_iter=False here, to reflect that the datasets are already iters